simulator/airportcomponent.py

```
@@ -105,19 +130,39 @@ def validate_message(self, required_keys: List[str], message: dict):
             return False
         return True
 
+    @abstractmethod
+    def on_child_connect(self):
+        """called by on_connect once connected to the mqtt broker"""
+
+    def on_connect(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)